moncli/entities/client.py

```diff
@@ -1273,7 +1281,337 @@ def create_workspace(self, name: str, kind: WorkspaceKind, *args, **kwargs):
+                            The user's creation date.
+                        email : `str`
+                            The user's email.
+                        enabled : `bool`
+                            Is the user enabled or not.
+                        id : `str`
+                            The user's unique identifier.
+                        is_guest : `bool`
+                            Is the user a guest or not.
+                        is_pending : `bool`
+                            Is the user a pending user.
+                        is_view_only : `bool`
+                            Is the user a view only user or not.
+                        join_date : `str`
+                            The date the user joined the account.
+                        location : `str`
+                            The user' location.
+                        mobile_phone : `str`
+                            The user's mobile phone number.
+                        name : `str`
+                            The user's name.
+                        phone : `str`
+                            The user's phone number.
+                        photo_original : `str`
+                            The user's photo in the original size.
+                        photo_small : `str`
+                            The user's photo in small size (150x150).
+                        photo_thumb : `str`
+                            The user's photo in thumbnail size (100x100).
+                        photo_thumb_small : `str`
+                            The user's photo in small thumbnail size (50x50).
+                        photo_tiny : `str`
+                            The user's photo in tiny size (30x30).
+                        teams : `list[moncli.entities.Team]`
+                            The teams the user is a member in.
+                        time_zone_identifier : `str`
+                            The user's time zone identifier.
+                        title : `str`
+                            The user's title.
+                        url : `str`
+                            The user's profile url.
+                        utc_hours_diff : `int`
+                            The user's UTC hours difference.
+
+
+        """
+        
+        users_data = api.delete_users_from_workspace(
+            workspace_id,
+            user_ids,
+            *args,
+            api_key=self.__creds.api_key_v2,
+            **kwargs)
+
+        return [en.User(creds=self.__creds, **data) for data in users_data]
+
+
+    def add_teams_to_workspace(self, workspace_id: str, team_ids: list(), *args, **kwargs ):
+
+        """Allows you to add teams to a workspace.
+
+        Parameters
 
+            workspace_id: str
+                The workspace's unique identifier.
+            team_ids: list()
+                Team ids to subscribe to the workspace.
+        
+        Returns
+
+            data : `dict`
```

Comment:
In this case, the return object is not a dictionary but instead a list of teams.  Please update.